client/src/Calendar.js

```diff
@@ -0,0 +1,198 @@
+            props.setRequest(userCalendar);
+          }).catch(reason => {
+            alert(reason)
+          });
+    }
+  }, [props.userName.id]);
+
+//states
+  const [isOpen, setOpen] = useState(false)
+
+  const [whatDate, setDate] = useState('')
+
+  const [whatTime, setWhatTime] = useState(7.5)
+
+  const [typeRadio, setType] = useState('sickday')
+
+  var today = new Date();
+
+// setting date to right format
+  today = today.toISOString().split('T')[0]
+  const todayTwo = today.split("-").join("/")
+
+// LOAD DATA from server to calendar **** EMPLOYEE ****
+  
+// LOAD DATA from server to calendar **** EMPLOYER ****
+
+// ********************* ADD EVENT - EMPLOYEE **************************
+
+const addEvent = async (e) => {
+  e.preventDefault();
+  
+  try {
+  // setting an object
+  const newDate = whatDate.split("-").join("/");
+
+  const dataAddEventEmployee = {
+    type: typeRadio === 'sickday' ? 'SICK_DAY' : 'VACATION',
+    date: newDate,
+    from: typeRadio === 'sickday' ? null : "00:00",
+    to: typeRadio === 'sickday' ? null : moment().startOf('day').add(whatTime, "hours").format("hh:mm"),
+  }
+
+  await api_fetch.addEventApi(dataAddEventEmployee);
+    if (typeRadio === 'holiday') {
+      props.setUserName({
+        ...props.userName,
+        holiday: props.userName.holiday - whatTime
+      })
+    } else if (typeRadio === 'sickday') {
+      props.setUserName({
+        ...props.userName,
+        takenSickday: props.userName.takenSickday + 1
+      })
+    }
+    
+    setOpen(false)
+  } catch (e) {
+    alert(e)
+  }
+}
+// ********************* ADD EVENT ADMIN - EMPLOYER **************************
+
+const addEventAdmin = async (e) => {
+  e.preventDefault();
+
+// setting an object
+  const newDate = whatDate.split("-").join("/");
+
+  const dataAddEventAdmin = {
+    type: typeRadio === 'sickday' ? 'SICK_DAY' : 'VACATION',
+    date: newDate,
+    from: typeRadio === 'sickday' ? null : "00:00",
+    to: typeRadio === 'sickday' ? null : moment().startOf('day').add(whatTime, "hours").format("hh:mm"),
+  };
+
+  api_fetch.addEventApiAdmin(dataAddEventAdmin).then(() => {
+    const userProps = {
+      title: props.userName.name, 
+      start: whatDate      
+  }
+  //concat new request to current ones
+      props.setRequest((acceptedRequest) => acceptedRequest.concat(userProps))
+  }).catch(reason => {
+      alert(reason)
+  });
+
+  setOpen(false)
+
+  api_fetch.getUsersOverview().then(usersOverview => {
+    props.setEmployees(usersOverview);
+    }).catch(reason => {
+    alert(reason)
+ });
+}
+    
+  return (
+    <div className="calendar">
+
+    <FullCalendar defaultView="dayGridMonth" plugins={[ dayGridPlugin, interactionPlugin ]}
+
+    dateClick={function(info) {
+      //setOpen(true === info.dateStr > today ? true : false )
+      setOpen(info.dateStr > today)
+      setDate(info.dateStr)
+      setWhatTime(7.5)
```

Comment:
Move `7.5` to a named constant, something like `const DEFAULT_MANDAY_HOURS = 7.5;`

Then use the constant here.